### ISSUE TITLE
error in frame-p010-out.jsonld

### ISSUE THREAD

**Issue by eroux** [json-ld/json-ld.org#676]
The `#tp010` test of frames specifies in the [manifest](https://github.com/json-ld/json-ld.org/blob/master/test-suite/tests/frame-manifest.jsonld#L540):

```json
{
      "@id": "#tp010",
      "@type": ["jld:PositiveEvaluationTest", "jld:FrameTest"],
      "name": "property CURIE conflict (prune bnodes)",
      "purpose": "(Not really framing) A term looking like a CURIE becomes a CURIE when framing/compacting if defined as such in frame/context.",
      "input": "frame-0010-in.jsonld",
      "frame": "frame-0010-frame.jsonld",
      "expect": "frame-p010-out.jsonld",
      "option": {"processingMode": "json-ld-1.1", "specVersion": "json-ld-1.1"}
    }
```

(note the `json-ld-1.1` processing mode) but expects the [following output](https://github.com/json-ld/json-ld.org/blob/master/test-suite/tests/frame-p010-out.jsonld):

```
{
  "@context": {
    "dc": "http://purl.org/dc/terms/",
    "dc:creator": {
      "@type": "@id"
    },
    "foaf": "http://xmlns.com/foaf/0.1/",
    "ps": "http://purl.org/payswarm#"
  },
  "@graph": [{
    "@id": "http://example.com/asset",
    "@type": "ps:Asset",
    "dc:creator": {
      "foaf:name": "John Doe"
    }
  }]
}
```

which I believe is wrong due to the omit graph flag being set when the processing mode is 1.1, and should instead be:

```json
result: {
  "@context" : {
    "dc" : "http://purl.org/dc/terms/",
    "dc:creator" : {
      "@type" : "@id"
    },
    "foaf" : "http://xmlns.com/foaf/0.1/",
    "ps" : "http://purl.org/payswarm#"
  },
  "@id" : "http://example.com/asset",
  "@type" : "ps:Asset",
  "dc:creator" : {
    "foaf:name" : "John Doe"
  }
}
```

or am I missing something? The same problem appears in (at least) `#tp046` and `#tp050`.

**Comment by gkellogg** [comment]
Yes, I think you're correct, although this will now go against https://github.com/w3c/json-ld-framing.

**Comment by gkellogg** [comment]
Closed in favor of w3c/json-ld-framing#10.
